### ISSUE TITLE
Double Click does not work in master

### ISSUE THREAD

**Issue by apriendeau** [neovim/neovim#4887]
- `nvim --version`: NVIM v0.1.5-386-g999590b
- VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Jun  6 2016 09:49:41)
- Operating system/version: OSX 10.11
- Terminal name/version: iterm2
- `$TERM`: xterm-256color

### Actual behaviour
Double click does not work after this release https://github.com/neovim/neovim/commit/8958863e2678fd038ae089f837fa46d19574f977

### Expected behaviour
Double click should still work as expected

### Steps to reproduce using `nvim -u NORC`
```
nvim -u NORC
set mouse=a
open file with contents: double click should select a word
```

**Comment by justinmk** [comment]
Thanks for bisecting. @sach1t would you be able to look at this?

**Comment by sach1t** [comment]
Okay. I will take a look at it tonight.

**Comment by sach1t** [comment]
Sorry for taking so long to get back to this. I found where the code that determines the multi-clicks is and I am just in the process of finding the cause of the issue. The only thing I need now is some free time :)

**Comment by sach1t** [comment]
Okay, so managed to fix the issue. I will be putting in a pull request soon :)

**Comment by sach1t** [comment]
This is now resolved and the patch is in master, so this issue can be closed.

**Comment by jamessan** [comment]
Fixed in 6da7d68.
